### PR TITLE
Align the Getting Started documentation with .NET 6. Fix #11888

### LINKED ISSUE #11888: We need to update the "Getting Started" section of the documentation
This section of the documentation is critically outdated, so much so that it is not possible to get the results that are described. 
As an example this [page](https://docs.orchardcore.net/en/dev/docs/getting-started/), the latest version of OrchardCore 1.4.0 which uses `.Net 6.0` and accordingly if you create a new empty project '.NET Core web application'  with `.Net 6.0`  you will not have a `Startap.cs` file in which all the basic manipulations should take place.

For new developers, this section is an important element of getting started with orchard. And such difficulties that in the guide can have a negative impact on entering a new environment.

My suggestion is to update this section for use on the latest stable version of OrchardCore and be sure to specify the platform versions on which it should work in case of obsolescence to be clear for which version this instruction is valid. Because at the moment there is only one[ page](https://docs.orchardcore.net/en/latest/docs/getting-started/templates/) of this section that mentions the versions for which this example is used.

### PR DESCRIPTION
**Pull request by CrestApps**
Fix #11888

### CODE REVIEW

File: src/docs/getting-started/README.md [hunk 1]
```diff
@@ -14,7 +14,61 @@ In Visual Studio, create a new empty .NET Core web application. Ex: `Cms.Web`. D
 
 To add a reference to the package, right-click on the project and click on `Manage NuGet packages...`, check `Include prerelease` if required. If you added the preview source above, select this from the `Package Source` selection in the top right.  In the `Browse` tab, search for `OrchardCore.Application.Cms.Targets` and `Install` the package.
 
-Open `Startup.cs` and modify the `ConfigureServices` method by adding this line:
+### Getting Started with `Program.cs` Only Using .NET 6 Framework?
+!!! tip
+    When starting a new project using `.NET 6` framework, you'll notice that the created project does not have a `Startup` class as it did in previous versions of the .NET framework.
+
+Open `Project.cs` file. Remove the following line "if exists"
```
Comment (hishamco) [inline src/docs/getting-started/README.md:21]: Project or Program?

File: src/docs/getting-started/README.md [hunk 2]
```diff
@@ -14,7 +14,61 @@ In Visual Studio, create a new empty .NET Core web application. Ex: `Cms.Web`. D
...
+    app.UseExceptionHandler("/Error");
+    app.UseHsts();
+}
+
+app.UseStaticFiles();
+app.UseOrchardCore();
+
+app.Run();
+```
+
```
Comment (hishamco) [inline src/docs/getting-started/README.md:67]: Extra line

File: src/docs/getting-started/README.md [hunk 3]
```diff
@@ -38,6 +92,8 @@ with this line:
 app.UseOrchardCore();
 ```
 
+Finally, remove the default `Pages` and/or `Views` folder to allow OrchardCore to render the views from the theme.
```
Comment (hishamco) [inline src/docs/getting-started/README.md:95]: from the **current/selected** theme.

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/docs/getting-started/README.md (hunk 3 above)
Comment (CrestApps) [reply src/docs/getting-started/README.md:94]: active theme?

**Review by hishamco — APPROVED** [review]

**Comment by CrestApps** [comment]
> Might be worth updating the [project templates](https://github.com/OrchardCMS/OrchardCore/tree/main/src/Templates/OrchardCore.ProjectTemplates/content) as well to use the .NET 6 minimal hosting model.

Good idea. I suggest you create a new issue for that.

**Comment by hishamco** [comment]
> Might be worth updating the [project templates](https://github.com/OrchardCMS/OrchardCore/tree/main/src/Templates/OrchardCore.ProjectTemplates/content) as well to use the .NET 6 minimal hosting model.

I think this tracked in another PR, so it's fine to merge this

**Pull request merged**